Docs: Update auditing docs with a basic authentication example (#75981)
grafana/grafana: docs/sources/setup-grafana/configure-security/audit-grafana.md

- **grafana_instance** - Application URL.
- **kind** - `auditing`

> When basic authentication is needed to ingest logs in your loki instance, you can specify credentials in the URL field. Here is an example:

Reviewer: ```suggestion
When basic authentication is needed to ingest logs in your Loki instance, you can specify credentials in the URL field. For example:
```
